From 1340ae7790b666d1f147bb6c019f7ee2a7ac1339 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Mon, 16 Nov 2009 09:58:06 +0800
Subject: [PATCH 131/435] ENGR00118445 VPU: Add STD_H263 for interlaced frame
 check in decoder

All STD_H263 processing is same as STD_MPEG4 in vpu decoder since
vpu treats H263 same as MPEG4.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 test/mxc_vpu_test/dec.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/mxc_vpu_test/dec.c b/test/mxc_vpu_test/dec.c
index 7d0c777..d09bf0c 100644
--- a/test/mxc_vpu_test/dec.c
+++ b/test/mxc_vpu_test/dec.c
@@ -945,7 +945,9 @@ decoder_start(struct decode *dec)
 					dprintf(3, "Top Field First flag: %d, dec_idx %d\n",
 						  outinfo.topFieldFirst, decIndex);
 				}
-			} else if ((dec->cmdl->format != STD_MPEG4) && (dec->cmdl->format != STD_RV)){
+			} else if ((dec->cmdl->format != STD_MPEG4) &&
+				   (dec->cmdl->format != STD_H263) &&
+				   (dec->cmdl->format != STD_RV)){
 				if (outinfo.interlacedFrame) {
 					if (outinfo.pictureStructure == 1)
 						field = V4L2_FIELD_TOP;
-- 
1.8.0

